src/components/SelectionList/BaseListItem.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {View} from 'react-native';
+import lodashGet from 'lodash/get';
+import PressableWithFeedback from '../Pressable/PressableWithFeedback';
+import styles from '../../styles/styles';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+import themeColors from '../../styles/themes/default';
+import {baseListItemPropTypes} from './selectionListPropTypes';
+import * as StyleUtils from '../../styles/StyleUtils';
+import UserListItem from './UserListItem';
+import RadioListItem from './RadioListItem';
+import OfflineWithFeedback from '../OfflineWithFeedback';
+import CONST from '../../CONST';
+
+function BaseListItem({item, isFocused = false, isDisabled = false, showTooltip, canSelectMultiple, onSelectRow, onDismissError = () => {}}) {
```

Comment:
I see that `canSelectMultiple` is not required.
```suggestion
function BaseListItem({item, isFocused = false, isDisabled = false, showTooltip, canSelectMultiple = true/false, onSelectRow, onDismissError = () => {}}) {
```